lib/aca_entities/families/family_member.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Families
+    class FamilyMember < Dry::Struct
+      # TODO: Person hbx_id until we add hbx_id on family member
+      attribute :hbx_id,                                      Types::String.optional.meta(omittable: false)
+      attribute :is_primary_applicant,                        Types::Bool.optional.meta(omittable: false)
+      attribute :is_consent_applicant,                        Types::Bool.optional.meta(omittable: false)
+      attribute :is_coverage_applicant,                       Types::Bool.optional.meta(omittable: false)
+      attribute :is_active,                                   Types::Bool.optional.meta(omittable: true)
+      attribute :applicant_reference,                         AcaEntities::MagiMedicaid::ApplicantReference.optional.meta(omittable: true)
+      attribute :former_family_reference,                     AcaEntities::Families::FamilyReference.optional.meta(omittable: true)
+      attribute :hbx_enrollment_exemptions,
```

Comment:
Remove hbx_enrollment_exemptions attribute until business case is verified.  

Is this associated with eligibility determination outcomes?  Is it to allow an exception to rules applied elsewhere?  Is it for IVL, SHOP, Congress, ALL?

Reply:
Looks like this associated with IVL, removed hbx_enrollment_exemptions, will add back after verifying  business case.